src/compiler/transformers/test/transpile.ts

```diff
@@ -15,53 +17,67 @@ import { getScriptTarget } from '../transform-utils';
  * @param compilerCtx a compiler context to use in the transpilation process
  * @param beforeTransformers TypeScript transformers that should be applied before the code is emitted
  * @param afterTransformers TypeScript transformers that should be applied after the code is emitted
+ * @param afterDeclarations TypeScript transformers that should be applied
+ * after declarations are generated
+ * @param tsConfig optional typescript compiler options to use
```

Comment:
I had to add these two options to 1) set the `afterDeclarations` transformers and 2) override some of the TS compiler options set before the transpilation happens.